Gopkg.toml

```diff
@@ -1,22 +1,7 @@
-
-# Gopkg.toml example
-#
+# Gopkg.toml
 # Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
 # for detailed Gopkg.toml documentation.
-#
-# required = ["github.com/user/thing/cmd/thing"]
-# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
-#
-# [[constraint]]
-#   name = "github.com/user/project"
-#   version = "1.0.0"
-#
-# [[constraint]]
-#   name = "github.com/user/project2"
-#   branch = "dev"
-#   source = "github.com/myfork/project2"
-#
-# [[override]]
-#  name = "github.com/x/y"
-#  version = "2.4.0"
 
+[[constraint]]
```

Comment:
Please put a comment here explaining the reason for the constraint and when it can be removed.

Reply:
We actually no longer need this constraint, since the initialisation no longer uses `CounterVec::Set`.
I'll remove this commit.